Core/ClientSMLSWIG/Python/SConscript

```diff
@@ -47,3 +53,8 @@ install_lib = env.Install(lib_install_dir, shlib)
 install_test = env.Install(lib_install_dir, env.File('TestPythonSML.py'))
 
 env.Alias(python_sml_alias, install_lib + install_source + install_test)
+
+python_shlib = shlib
+python_source = source
+Export('python_shlib')
+Export('python_source')
```

Comment:
It exports these variables, the first defines them under a different name, and the latter two exports them to a global table that is then pulled from with `Import` in the main SConstruct file.

There, the `Whl` part needs to be defined, it will not work in any SConscript script, notably not this one, unfortunately.

The `Whl` needs the source file / build definitions to pull from and trigger builds of, so this is just airlifted out of here, into that Whl definition